programs/config/src/config_processor.rs

```diff
@@ -96,7 +96,7 @@ pub fn process_instruction(
         return Err(InstructionError::InvalidInstructionData);
     }
 
-    config_keyed_account.try_account_ref_mut()?.data[0..data.len()].copy_from_slice(&data);
+    config_keyed_account.try_account_ref_mut()?.data[..].copy_from_slice(&data);
```

Comment:
Looks like this change broke a bunch of tests.

Reply:
I have trouble figuring out [why](https://github.com/rust-lang/rust/blob/master/src/libcore/ops/range.rs#L26-L36). I suspect a dependency interaction.

Reply:
`data != config_keyed_account.try_account_ref_mut()?.data`

Reply:
Deleting just the zero from the original code would obfuscate things a little less.

Reply:
:man_facepalming: